Sources/StripeKit/Billing/Invoices/Invoice.swift

```diff
@@ -318,6 +320,11 @@ public struct Invoice: Codable {
     }
 }
 
+public struct SubscriptionDetails: Codable {
+	/// Set of key-value pairs that you can attach to an object. This can be useful for storing additional information about the object in a structured format.
+	public var metadata: [String: String]?
```

Comment:
@Maxim-Inv Can you add a public initializer for this type?

Reply:
Yes